converter/include/k4SimDelphes/DelphesEDM4HepOutputConfiguration.h

```diff
@@ -19,7 +19,7 @@ namespace k4SimDelphes {
      * under the name specified below. These will be associated with the generated
      * particles.
      */
-    std::vector<std::string> ReconstructedParticleCollections{{"EflowTrack", "EFlowPhoton", "EFlowNeutralHadron"}};
+    std::vector<std::string> ReconstructedParticleCollections{{"EflowTrack", "EFlowPhoton", "EFlowNeutralHadron", "ParticleFlowCandidate"}};
```

Comment:
If I understood correctly, they should not be part of the "default" collections, right? If that is the case they should be removed here again.